rdkit/SimDivFilters/UnitTestSimilarityPickers.py

```diff
@@ -0,0 +1,114 @@
+# $Id$
```

Comment:
Also, it looks like this file is just a duplicate of the doctests that are in `SimilarityPickers.py`. That doesn't make any sense to me. Just sent an email about this.